scripts/profiler/rtt_nordic_profiler_host.py

```diff
@@ -32,22 +32,30 @@ def __init__(self, config=RttNordicConfig, finish_event=None,
         self.received_events = EventsData([], {})
         self.timestamp_overflows = 0
         self.after_half = False
+
+        self.desc_buf = ""
+        self.bufs = list()
+        self.bcnt = 0
+        self.last_read_time = time.time()
+        self.reading_data = True
+
         self.logger = logging.getLogger('RTT Profiler Host')
         self.logger_console = logging.StreamHandler()
         self.logger.setLevel(log_lvl)
         self.log_format = logging.Formatter('[%(levelname)s] %(name)s: %(message)s')
         self.logger_console.setFormatter(self.log_format)
         self.logger.addHandler(self.logger_console)
-        self._connect()
+        
+        self.connect()
 
     def rtt_get_device_family():
         with API.API(API.DeviceFamily.UNKNOWN) as api:
             api.connect_to_emu_without_snr()
             return api.read_device_family()
 
-    def _connect(self):
+    def connect(self):
         try:
-            self.jlink = API.API(self.config['device_family'])
+            self.jlink = API.API(self.config['device_family'], jlink_arm_dll_path='JLinkARM.dll')
```

Comment:
What is the effect of specifying the DLL name? Were there problems with finding the JLink DLL?

Reply:
Removed